docs/CppInteroperability/Roadmap.md

```diff
@@ -0,0 +1,141 @@
+# Swift and C++ Interoperability Roadmap
+
+* Introduction
+    * C++, as a language
+    * C and Objective-C interoperability
+* Specifics and trade-offs
+* Goals
+* The approach 
```

Comment:
Does it make sense to provide some examples of how we import various APIs? Maybe as an appendix at the end?

Reply:
It would be very helpful for me, and would help solidify a lot of the vocabulary.

Reply:
I agree as well. I far too often have to help folks experimenting with C++-Interop to get swift-ide-test built just for them to see some examples like these actually.